yogstation/code/modules/scripting/IDE.dm

```diff
@@ -49,7 +49,10 @@
 					if(!telecomms_check(mob))
 						return
 
-					if(compileerrors.len)
+					if(!compileerrors)
+						src << output("<b>Memory Error:</b> Script ran out of memory. Please ensure that the script does not have a large file size and contains no special characters.", "tcserror")
```

Comment:
What you're referring to is probably this. compilerrors returns null if the script contains bad characters. Checking null.len causes a runtime as seen in one of the screenshots above. if a script is successful, compileerrors will return list().

Reply:
It also returns null if it fails for any other reason

Reply:
Yep. The "Out of Memory" still seems applicable for both those situations.

Reply:
It isn’t

Reply:
I really don't mean to be rude but I think you need to be more clear in what you want and what is actually going on so it would save me more time and also save you more time.